docs/examples/params/model.yaml

```diff
@@ -0,0 +1,317 @@
+# Type of model that the rest of these parameters apply to.
```

Comment:
I see, yeah might be fine here. I think the real source of truth should be the algorithm doc strings but those are spread out throughout the code so this can be the place where we include them all. We may want to have references back to the individual algorithms in case they get out of sync.